doc/README.librexgen

```diff
@@ -82,22 +88,52 @@ https://github.com/teeshop/rexgen
 
 git clone --recursive https://github.com/teeshop/rexgen.git
 
-As of this writing the last known good version is 2.0.8 so you may want
-to check that out.
 
-Once you download it, it requires a cmake environment, along with flex and
-bison to be installed and working.  To build, see instructions on this page:
+As of this writing there is no known good rexgen version.
+
+The latest commit as of this writing is
+
+    commit 585e86da6a07a63ec7a871320b307457661d557c
+    Author: Jan Starke <teeshop@users.noreply.github.com>
+    Date:   Thu Apr 11 08:38:41 2019 +0200
+
+        Update README.md
+
+The latest known good commit is
+
+commit 5b2f4b159ec948c1f9429eca4389ca2adc9c0b07
+Author: Jan Starke <jan.starke@t-systems.com>
+Date:   Wed Jan 23 14:53:50 2019 +0100
+
+    running with libFuzzer works
+
+
+Checkout this latest known good commit:
+
+cd rexgen
+git checkout 5b2f4b159ec948c1f9429eca4389ca2adc9c0b07
+
+
+
+Rexgen requires a cmake environment, along with flex and bison
+to be installed and working.  To build, see instructions on this page:
```

Comment:
Maybe replace the first sentence with simply "Building rexgen requires cmake, flex, and bison."